lib/seadsa/DsaLocal.cc

```diff
@@ -468,36 +479,59 @@ class IntraBlockBuilder : public InstVisitor<IntraBlockBuilder>,
   void visitAllocWrapperCall(CallSite &CS);
   void visitAllocationFnCall(CallSite &CS);
 
+  const llvm::StringRef m_seaFnModTag       = "sea_dsa_set_modified";
+  const llvm::StringRef m_seaFnReadTag      = "sea_dsa_set_read";
+  const llvm::StringRef m_seaFnPtrToIntTag  = "sea_dsa_set_ptrtoint";
+  const llvm::StringRef m_seaFnAliasTag     = "sea_dsa_alias";
+  const llvm::StringRef m_seaFnCollapseTag  = "sea_dsa_collapse";
+  const llvm::StringRef m_seaFnMkSeqTag     = "sea_dsa_mk_seq";
+
+  SeadsaFn getSeaDsaFn (const Function *fn) {
+    if (!fn)  return SeadsaFn::UNKNOWN;
+
+    if (fn->getName().equals(m_seaFnModTag))      return SeadsaFn::ALIAS;
```

Comment:
FYI 1, you can use `==` to compare a StringRef with other string types (StringRef, StringLiteral, std::string, cstrings)
FYI 2, this seems like a perfect use case for StringSwitch: https://llvm.org/doxygen/classllvm_1_1StringSwitch.html, as you can turn all tags into llvm::StringLiteral

Reply:
Done in 60a272f. I was unaware of StringSwitch but I agree that it's great here